### PR TITLE
[tune] Doc fixes

### PR DESCRIPTION
**Pull request by adizim**

### CODE REVIEW

File: doc/source/tune-searchalg.rst [hunk 1]
```diff
@@ -130,8 +131,8 @@ An example of this can be found in `nevergrad_example.py <https://github.com/ray
     :show-inheritance:
     :noindex:
 
-Scikit-Optimize Search
```
[the remark this answers is not in the document]
Comment (adizim) [reply doc/source/tune-searchalg.rst]: Sure, would the bullet link to it just be <tune-searchalg.html#scikit-optimize-search> in that case?

[the remark this answers is not in the document]
Comment (richardliaw) [reply doc/source/tune-searchalg.rst]: Oh, you can try these things out by running 
`make html` in `ray/doc` -- can you let me know if it works?

[the remark this answers is not in the document]
Comment (adizim) [reply doc/source/tune-searchalg.rst]: Thanks, it works

**Review by richardliaw — APPROVED** [review]

File: doc/source/tune-searchalg.rst [hunk 2]
```diff
@@ -16,6 +16,7 @@ Currently, Tune offers the following search algorithms (and library integrations
 - `HyperOpt <tune-searchalg.html#hyperopt-search-tree-structured-parzen-estimators>`__
 - `SigOpt <tune-searchalg.html#sigopt-search>`__
 - `Nevergrad <tune-searchalg.html#nevergrad-search>`__
+- `SkOpt <tune-searchalg.html#scikit-optimize-search>`__
```
Comment (richardliaw) [inline doc/source/tune-searchalg.rst]:
```suggestion
- `Scikit-Optimize <tune-searchalg.html#scikit-optimize-search>`__
```

Comment (richardliaw) [reply doc/source/tune-searchalg.rst]: (this should improve SEO)

**Pull request merged**